exercises/concept/regular-chatbot/.meta/config.json

```diff
@@ -0,0 +1,11 @@
+{
+  "blurb": "Learn the basics of Regular Expressions with JavaScript helping a Chatbot understand humans.",
```

Comment:
```suggestion
  "blurb": "Learn the basics of regular expressions in JavaScript helping a chatbot understand humans.",
```

Reply:
As said earlier, I'll apply this change manually, just in case. Thanks for the correction, though.